src/service/performance-impl.js

```diff
@@ -214,7 +214,7 @@ export class Performance {
     const data = {
       label,
       value,
-      delta: opt_delta,
+      delta: opt_delta != null ? Math.max(opt_delta, 1) : undefined,
```

Comment:
Comment, and add comment to `tickDelta` that delta can be negative, but will be changed to 1.